testdata/project-v4-with-plugins/internal/controller/memcached_controller.go

```diff
@@ -31,6 +31,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/record"
+	"k8s.io/utils/ptr"
```

Comment:
Why we need a new dep?
I thought if we were here very surgically just change the place that would be better to create a New Error and return it instead of using requeue true